pkgs/development/tools/gojsontoyaml/default.nix

```
@@ -0,0 +1,22 @@
+{ stdenv, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "gojsontoyaml";
+  version = "unstable-2020-06-02";
+
+  src = fetchFromGitHub {
+    rev = "${version}";
+    owner = "brancz";
+    repo = "gojsontoyaml";
+    sha256 = "07sisadpfnzbylzirs5ski8wl9fl18dm7xhbv8imw6ksxq4v467a";
```

```suggestion
    owner = "brancz";
    repo = "gojsontoyaml";
    rev = version;
    sha256 = "07sisadpfnzbylzirs5ski8wl9fl18dm7xhbv8imw6ksxq4v467a";
```

Also this will not work. Please tag a commit if upstream has not tags.

---

Done. Sorry for the mistake.